ports/knet/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "knet",
-  "version-string": "1.0.1",
-  "port-version": 1,
+  "version-string": "1.0.2",
+  "port-version": 2,
```

```suggestion
  "version": "1.0.2",
  "port-version": 0,
```

Since the version string has changed the `port-version` field must be reset to `0`. Also, this should be [using `version` or `version-semver` instead of `version-string`](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#schemes).